### DISCUSSION TITLE
[Discussion]: 关于媒体库命名问题

### DISCUSSION THREAD

**Discussion by abbcbbdd** [jxxghp/MoviePilot#376] [讨论]
### 项目讨论

个人的目录结构是media文件夹下电影、电视剧、最新电影、最新电视剧、Hlink，Hlink下是电影、电视剧、最新电影、最新电视剧，其中电影、电视剧两个文件夹用于放置个人手动整理压制的影片，最新电影和最新电视剧则放置MP订阅的电影，看完觉得不错再手动调整压制放到电影、电视剧目录，这样收藏的电影和订阅的电影分开以便于管理，但是LIBRARY_MOVIE_NAME和LIBRARY_TV_NAME两个参数强制添加文件夹，导致设置Hlink为媒体库时，目录变成了media-Hlink-最新电影-电影-XXX，多了一个层级，希望能给个选项决定是否添加电影、电视剧文件夹，手动整理时完全可以配合目录监控插件自己定义目录名称，可以不再多一层级，谢谢！

**Comment by dlovew** [comment]
LIBRARY_MOVIE_NAME=最新电影即可

**Comment by abbcbbdd** [comment]
> LIBRARY_MOVIE_NAME=最新电影即可

那电影目录就会出现media-hlink-电影-最新电影-xxx，其实就是有两个目录，一个收藏目录，一个下载目录，两个目录名字不一样，无论参数命名什么都会有不对的，就是想着可不可以给个选项直接去掉一级命名，一级命名的电影、电视剧完全可以自己创建目录来实现，mp直接分二级目录就行了
